### ISSUE TITLE
User disabled

### ISSUE THREAD

**Issue by Teste55** [FastCopyLab/ipmsg#44]
Good morning,

I went to upgrade a company computer from HD to SSD and I no longer have access to the HD, when installing the application again, the user logged in to the HD appears as if he were offline, it is possible to remove this user through another account connected to the network or even, merge this user with another existing one?

**Comment by shirouzu** [comment]
I'm sorry, but I don't understand the situation very well.
Please explain more clearly and in detail.

**Comment by Teste55** [comment]
What happened is that I had to do a hardware update, and I ended up losing the HD I was using previously. When installing an SSD to meet this need, I did all the necessary installations and logged back into IPMSG.

What happens is that the user I used continues to appear in the list of users connected to the network (as if it were offline), I would like to know if there is a way to delete this user from the list (For all computers connected to this network). Remembering that I can no longer directly access the computer that had this user, now unwanted, anymore
